PG-208 Doc Added Contibuting guide (#103)
percona/pg_stat_monitor: CONTRIBUTING.md

> # Contributing guide

Reviewer: ```suggestion
# Contributing Guide
```

> Welcome to `pg_stat_monitor` - the statistics collection tool for PostgreSQL!

Reviewer: ```suggestion
Welcome to `pg_stat_monitor` - the Query Performance Monitoring Tool for PostgreSQL!
```

Author: Fixed

We're glad that you would like to become a Percona community member and participate in keeping open source open.

You can contribute in one of the following ways:

> 

Reviewer: Lots of empty lines in this document, could you trim these down?

Author: Done

1. Reach us on our [Forums](https://forums.percona.com/) and Discord.
2. [Submit a bug report or a feature request](#submit-a-bug-report-or-a-feature-request)
3. [Submit a pull request (PR) with the code patch](#submit-a-pull-request)
4. [Contribute to documentation](#contributing-to-documentation)

By contributing, you agree to the [Percona Community code of conduct](https://github.com/percona/community/blob/main/content/contribute/coc.md).

## Submit a bug report or a feature request

> If you found a bug in `pg_stat_monitor`, please submit the report to the [Jira issue tracker](https://jira.percona.com/projects/PG/issues).

Reviewer: ```suggestion
If you would like to suggest a feature of if you found a bug in `pg_stat_monitor`, please submit the report to the [Jira issue tracker](https://jira.percona.com/projects/PG/issues).
```

Author: Fixed

Start by searching the open tickets for a similar report. If you find that someone else has already reported your issue, then you can upvote that report to increase its visibility.

If there is no existing report, submit your report following these steps:

1. Sign in to [Jira issue tracker](https://jira.percona.com/projects/PG/issues). You will need to create an account if you do not have one.
2. In the _Summary_, _Description_, _Steps To Reproduce_, _Affects Version_ fields describe the problem you have detected or an idea that you have for a new feature or improvement.
3. As a general rule of thumb, try to create bug reports that are:

  * Reproducible: describe the steps to reproduce the problem.
  * Unique: check if there already exists a JIRA ticket to describe the problem.
  * Scoped to a Single Bug: only report one bug in one JIRA ticket

## Submit a pull request

> All bug reports, enhancements and feature requests are tracked in [Jira issue tracker](https://jira.percona.com/projects/PG). Though not mandatory, we encourage you to first check for a bug report among Jira issues and in the PR list: perhaps the bug has already been addressed.

Reviewer: This paragraph should be moved up, into the "Submit a bug or feature request" section.

Author: Amended

5. Test your changes locally. See the [Running tests ](#running-tests) section for more information
6. Update the documentation describing your changes. See the [Contributing to documentation](#contributing-to-documentation) section for details
8. Commit the changes. Add the Jira issue number at the beginning of your message subject, so that is reads as `<JIRAISSUE> : My commit message`.  Follow this pattern for your commits:

    ```
    PG-1234:  Main commit message.
    <Blank line>
    Details of fix.
    ```

    The [commit message guidelines](https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53) will help you with writing great commit messages

9. Open a pull request to Percona
10. Our team will review your code and if everything is correct, will merge it. Otherwise, we will contact you for additional information or with the request to make changes.
11. Make sure your pull request contains only one commit message

### Building pg_stat_monitor

To build `pg_stat_monitor` from source code, you require the following:

* git
* make

> 

Reviewer: There's probably more to it, e.g. a C compiler? @darkfronza or @ibrarahmad - can you add some more details here?
